core/modules/layout/layout.admin.inc

```diff
@@ -1234,7 +1234,7 @@ function layout_content_form($form, &$form_state, Layout $layout, $renderer_name
 }
 
 /**
- * Validate the layout settings form.
```

Comment:
@klonos There are other layout settings that we will be adding here. Let's not change all the code comments now, and then need to change them back.